### DISCUSSION TITLE
No drag affordance for dialogs

### DISCUSSION THREAD

**Discussion by moonmehta** [elementary/stylesheet#1001] [General]
The screenshot tool window doesn't give any visual clue that the window can be dragged like any other window. It doesn't even change the cursor icon to the "hand" one to indicate the same. 

![Screenshot from 2019-05-14 19-55-22](https://user-images.githubusercontent.com/22585732/57706430-5a57af00-7683-11e9-83d1-704f944d8aa3.png)

The exact same issue is also seen in the file transfer dialog.

**Comment by peteruithoven** [comment]
I don't think any app uses a hand cursor to indicate "draggablity". 
Could you clarify what you mean with "like any other"?

**Comment by moonmehta** [comment]
I mean that the screenshot tool window doesn't give any visual clue that the window can be dragged like _**any other window**_.

**Comment by peteruithoven** [comment]
(The term for this is an "affordance": https://www.interaction-design.org/literature/topics/affordances)

**Comment by fitojb** [comment]
@elementary: I think this issue should be transferred to elementary/stylesheet, since the Screenshot tool isn’t the only affected window.
